File: FE/socialV/src/app/Model/Post-display.ts
// @ts-ignore
import { Users} from "./Users";
import {PostStatus} from "./post-status";

export interface PostDisplay {
  id?:number,
  users?:Users,
  content?:string,
  createAt?:Date,
  postStatus?:PostStatus,
  checkUserLiked?:boolean
}

File: BE/SocialV/src/main/java/com/example/socialv/dto/PostDisplay.java
package com.example.socialv.dto;

import com.example.socialv.model.PostStatus;
import com.example.socialv.model.Users;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PostDisplay {
    private Long id;
    private Users users;
    private String content;
    private PostStatus postStatus;
    private LocalDateTime createAt;
    private boolean checkUserLiked;
}
